test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.AttributesWithBooleanValues.html

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
It's not. It's collateral from @ajaybhargavb not running the functional tests with generate baselines prior to checking in. I let him know.